### PR TITLE
pool_stable: adds more contracterrrors

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

File: contracts/pool/src/contract.rs [hunk 1]
```diff
@@ -597,7 +597,7 @@ impl LiquidityPoolTrait for LiquidityPool {
         } else if offer_asset == config.token_b {
             (pool_balance_b, pool_balance_a)
         } else {
-            log!(&env, "Token offered to swap not found in Pool");
+            log!(&env, "Pool: Token offered to swap not found in Pool");
```
Comment (ueco-jb) [inline contracts/pool/src/contract.rs:600]:
```suggestion
            log!(&env, "Pool stable: Token offered to swap not found in Pool");
```

Repeat in all.

File: contracts/pool/src/storage.rs [hunk 2]
```diff
@@ -63,7 +63,7 @@ pub fn validate_fee_bps(env: &Env, total_fee_bps: i64) -> i64 {
         );
         panic_with_error!(
             env,
-            ContractError::ValidateFeeBpsTotalFeesCantBeGreaterThen100
+            ContractError::ValidateFeeBpsTotalFeesCantBeGreaterThan100
```
Comment (ueco-jb) [inline contracts/pool/src/storage.rs:66]: Good find. :)

**Review by ueco-jb — COMMENTED** [review]
Minor things.

File: contracts/pool/src/contract.rs (hunk 1 above)
Comment (gangov) [reply contracts/pool/src/contract.rs:600]: done

**Comment by ueco-jb** [comment]
> does that need to be addressed

Yes.

**Comment by gangov** [comment]
> > does that need to be addressed
> 
> Yes.

alright, I'll do it in this PR

**Review by ueco-jb — APPROVED** [review]
Great work, LGTM

**Pull request merged**